packages/opentelemetry-node/test/spanprocessor.test.ts

```diff
@@ -22,7 +26,13 @@ describe('SentrySpanProcessor', () => {
   let spanProcessor: SentrySpanProcessor;
 
   beforeEach(() => {
-    hub = new Hub();
+    const client = new NodeClient({
```

Comment:
@AbhiPrasad I see here: https://github.com/getsentry/sentry-javascript/pull/6114/files#diff-c10f2a296bc2f7b1e2908434ede6d48588c8add094f6270465f06fd41b35ac34R85 you added a mock client, should we do the same here? I wasn't quite sure.

Reply:
I don't mind either way - using a mock client was faster for me, but I'm fine with creating a NodeClient here.